components/navbar.js

```diff
@@ -0,0 +1,75 @@
+import { useTheme } from "next-themes";
+import Link from "next/link";
+import { useEffect, useState } from "react";
+
+export default function Navbar() {
```

Comment:
The file name should match the function name you exporting.. and should start in caps